modules/op-using-privileged-builds.adoc

```diff
@@ -0,0 +1,64 @@
+==== Running pods without privileged security context (default)
+
+The default configuration of OpenShift Pipelines 1.3.x and 1.4.x  does not allow you to run pods  with privileged security context, if the pods result from pipeline run or task run.
+For such pods, the default service account is `pipeline`, and the Security Context Constraint (SCC) associated with the `pipelines` service account is `pipelines-scc`. The `pipelines-scc` service account is similar to `anyuid`, but with a minor difference as defined in the YAML file for SCC of pipelines:
```

Comment:
`pipelines-scc` is an "security context constraint (SCC)", not a service account